pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -881,8 +908,9 @@ func (os *OpenStack) getVolume(diskName string) (volumes.Volume, error) {
 	return volume, err
 }
 
+// getComputeIDbyHostname tries to find ID of the OpenStack instance where
+// this code runs.
 func (os *OpenStack) getComputeIDbyHostname(cClient *gophercloud.ServiceClient) (string, error) {
```

Comment:
@spothanis, I think this is your code, what's your opinion? I am in favor of replacing it with simple `getServerByHostName(ossys.Hostname())` call.